e2e_tests/tests/experiment/test_pytorch.py

```diff
@@ -9,13 +10,24 @@
 
 @pytest.mark.e2e_gpu
 @pytest.mark.parametrize("aggregation_frequency", [1, 4])
+@pytest.mark.parametrize("image_type", ["PT", "TF2"])
 def test_pytorch_11_const(
-    aggregation_frequency: int, using_k8s: bool, collect_trial_profiles: Callable[[int], None]
+    aggregation_frequency: int,
+    image_type: str,
+    using_k8s: bool,
+    collect_trial_profiles: Callable[[int], None],
 ) -> None:
     config = conf.load_config(conf.fixtures_path("mnist_pytorch/const-pytorch11.yaml"))
     config = conf.set_aggregation_frequency(config, aggregation_frequency)
     config = conf.set_profiling_enabled(config)
 
+    if image_type == "PT":
```

Comment:
Nitpick: I think it is good as is but an alternative idea would be to parametrize with `("use_pytorch_only_image", [True, False])`. That would simplify the if statement, obviating the need for the `else` case.

Reply:
Makes sense. After better understanding the way e2e_tests work, I think an even better solution could have been not using parametrize at all and instead adding a new test set via something like `@pytest.mark.no_tf` and updating .circleci accordingly.